dhall-toml/src/Dhall/Toml.hs

```diff
@@ -25,68 +29,134 @@ import qualified Dhall.Parser
 import qualified Dhall.TypeCheck
 import qualified Toml.Type.TOML     as Toml.TOML
 import qualified Toml.Type.Value    as Toml.Value
+import qualified Toml.Type.AnyValue as Toml.AnyValue
 
 
 data CompileError
     = Unimplemented String
     | Unsupported (Expr Void Void)
+    -- tomland does not support records in multi-dimensional arrays, though it
```

Comment:
This might be useful to include in the rendered docs.

```suggestion
    -- | tomland does not support records in multi-dimensional arrays, though it
```